pages/dkp/kommander/2.2/dkp-upgrade/custom-apps/index.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: layout.pug
+navigationTitle: Upgrade Custom Applications
+title: Upgrade Custom Applications
+menuWeight: 60
+excerpt: Verify the compatibility of Custom Applications with the current Kubernetes version
+beta: false
+enterprise: true
+---
+
+We recommend upgrading your Custom Applications to the latest compatible version as soon as possible. Since there is no expectation of support by D2iQ for Custom Applications, you must upgrade them manually.
+
+<p class="message--warning"><strong>WARNING: </strong>Ensure you validate any Custom Applications you run for compatibility issues against the [current Kubernetes version][release-notes]. If the Custom Application’s version is not compatible with the Kubernetes version, do not continue with the Konvoy upgrade. Otherwise, your cluster’s services will stop running.
```

Comment:
The links here for `current Kubernetes version` and `release-notes` seem broken.

Reply:
Likely because it's in the html wrapper, dolbey had a fix for this, i'll look into it and add a commit.

Reply:
Maybe just switching to html completely? However, we won't have a relative link: 
```suggestion
<p class="message--warning"><strong>WARNING: </strong>Ensure you validate any Custom Applications you run for compatibility issues against the <a href="https://docs.d2iq.com/dkp/kommander/2.2/release-notes/">current Kubernetes version</a>. If the Custom Application’s version is not compatible with the Kubernetes version, do not continue with the Konvoy upgrade. Otherwise, your cluster’s services will stop running.
```